include/ddc/kernels/splines/bsplines_non_uniform.hpp

```diff
@@ -75,23 +85,20 @@ class NonUniformBSplines : detail::NonUniformBSplinesBase
         int m_nknots;
 
     public:
+        /// @brief The type of discrete dimension representing the BSplines.
         using discrete_dimension_type = NonUniformBSplines;
```

Comment:
```suggestion
        /// @brief The type of the discrete dimension representing the B-splines.
        using discrete_dimension_type = NonUniformBSplines;
```
This change also needs applying to the next 3 docsctrings. "the" or "a" is needed for non-plural nouns. "du" = "of the", "d'un" = "of a"